modules/oadp-about-bsl.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+[id="oadp-about-bsl_{context}"]
+= Configuring backup locations
+
+The OpenShift API for Data Protection (OADP) backs up Kubernetes objects and internal images by using Velero backups and storing them in backup locations, which are S3 storage endpoints.
+
+You configure backup locations in the `spec.backupLocations.velero` block of the `DataProtectionApplication` CR.
+
+[IMPORTANT]
+====
+You must use either a dedicated bucket for Velero or a bucket prefix, if the bucket is used for multiple purposes.
+====
+
+.Example spec.backupLocations.velero block
+ifdef::installing-oadp-aws[]
+[source,yaml]
+----
+spec:
+  backupLocations:
+    - name: default
+      velero:
+        provider: aws
+        default: true
+        credential:
+          name: cloud-credentials
+          key: cloud
+        config:
+          region: us-east-2
+          profile: "default"
+        objectStorage:
+          bucket: my_bucket
+          prefix: velero
+----
+endif::[]
+ifdef::installing-oadp-azure[]
+[source,yaml]
+----
+spec:
+  backupLocations:
+  - config:
+      resourceGroup: Velero_Backups
+      storageAccount: velerobackups
+      subscriptionId: 53b8f551-f0fc-4bea-8cba-6d1fefd54c8a
+      storageAccountKeyEnvVar: AZURE_STORAGE_ACCOUNT_ACCESS_KEY
+    provider: azure
+    default: true
+    credential:
+      name: cloud-credentials
+      key: cloud
+    objectStorage:
+      bucket: my_bucket
+      prefix: velero
+----
+endif::[]
+ifdef::installing-oadp-gcp[]
+[source,yaml]
+----
+spec:
+  backupLocations:
+    - velero:
+        provider: gcp
+        default: true
+        credential:
+          name: cloud-credentials
+          key: cloud
+        objectStorage:
+          bucket: my_bucket
+          prefix: velero
+----
+endif::[]
+ifdef::installing-oadp-mcg[]
+[source,yaml]
+----
+spec:
+  backupStorageLocations:
```

Comment:
The block `backupStorageLocations` needs update, it's not the latest. please reference https://docs.google.com/document/d/1YkEQLmTVu4lS88xmoyQLAxYVm-BvrDusemPJptclvjQ/edit#heading=h.bj376sqguyfz

Reply:
This is for MCG. It is not updated in your doc.

Reply:
Or do you mean that I could use the S3? Looks like it would work.